projects/demo/src/modules/app/app.config.ts

```diff
@@ -252,5 +253,9 @@ export const config: ApplicationConfig = {
             eventCoalescing: false,
             runCoalescing: false,
         }),
+        {
+            provide: REMOVE_STYLES_ON_COMPONENT_DESTROY,
+            useValue: false,
```

Comment:
Do we need it here if `TuiRoot` already provides it ?

Reply:
I was using it for tests by providing `true` here because that's the default in newer versions for Angular. Let's keep it, I just forgot to switch it back to `true`